src/api/routes/test.ts

```diff
@@ -0,0 +1,92 @@
+import AsyncRouter from '../AsyncRouter';
+import { Response } from 'express';
+import {
+  createOrUpdateTest,
+  getUser,
+  getTests,
+} from '../../application/service';
+import { ApiError } from '../ApiError';
+import { User } from '../../domain/model/user/User';
+//import {Test as ApiTest} from '../interface/index';
+import { Test } from '../../domain/model/test/Test';
+
+import logger from '../../logger';
+import { isAuthenticated } from '../middleware/isAuthenticated';
+import { AuthenticatedRequest } from '../AuthenticatedRequest';
+
+export default () => {
+  const route = new AsyncRouter();
+
+  route.get(
+    '/users/:id/tests',
+    isAuthenticated,
+    async (req: AuthenticatedRequest, res: Response) => {
+      const { id } = req.params;
+      const user = await getUser.byId(id);
+
+      if (!user) {
+        throw new ApiError(404, 'user.not-found');
+      }
+
+      const tests = await getTests.byUserId(id);
```

Comment:
We'll need to integrate permissions here, like we do around this section: https://github.com/cov-clear/backend/blob/master/src/api/routes/user.ts#L42